test/broker_test/broker_test.go

```diff
@@ -1087,44 +1189,55 @@ var _ = test.DescribeTestsFor(test.TestCase{
 									Status(http.StatusConflict).JSON().Object().Keys().Contains("error", "description")
 
 								assertInvocationCount(brokerServer.CatalogEndpointRequests, 1)
+							})
+
+							It("is returned from the repository in the brokers catalog field", func() {
+								ctx.SMWithOAuth.PATCH("/v1/service_brokers/" + brokerID).
```

Comment:
This code is pretty similar and could probably be extracted into something like this:


```go
func verifyRepositoryReturnsBrokerCatalogField(brokerID, expectedCatalog string, expectedStatus int) {
        ctx.SMWithOAuth.PATCH("/v1/service_brokers/" + brokerID).
	    WithJSON(common.Object{}).
	    Expect().
	    Status(expectedStatus)

 	brokerFromDB, err := repository.Get(context.TODO(), types.ServiceBrokerType, brokerID)
	Expect(err).ToNot(HaveOccurred())

 	Expect(string(brokerFromDB.(*types.ServiceBroker).Catalog)).To(MatchJSON(expectedCatalog))
}
```